codex/stores/repostore/operations.nim

```diff
@@ -162,15 +160,15 @@ proc storeBlock*(self: RepoStore, blk: Block, minExpiry: SecondsSince1970): Futu
         else:
           raise newException(CatchableError, "Repo already stores a block with the same cid but with a different size, cid: " & $blk.cid)
       else:
-        md = BlockMetadata(size: blk.data.len.NBytes, expiry: minExpiry, refCount: 0)
+        md = BlockMetadata(size: blk.data.len.NBytes, refCount: 0)
         res = StoreResult(kind: Stored, used: blk.data.len.NBytes)
         if err =? (await self.repoDs.put(blkKey, blk.data)).errorOption:
           raise err
 
       (md.some, res)
   )
 
-proc tryDeleteBlock*(self: RepoStore, cid: Cid, expiryLimit = SecondsSince1970.low): Future[?!DeleteResult] {.async.} =
+proc tryDeleteBlock*(self: RepoStore, cid: Cid): Future[?!DeleteResult] {.async.} =
   if cid.isEmpty:
```

Comment:
The fact that metadata and block state are not updated transactionally makes me a bit anxious. I can see for instance that there is room for metadata to be left behind after a block deletion.

Reply:
Not sure how to address this.